utils/utils.py

```diff
@@ -0,0 +1,16 @@
+import logging
+import sys
+
+class Utils:
+    def __init__(self):
+
+        self.logger = logging.getLogger()
+        self.streamHandler = logging.StreamHandler(sys.stdout)
+        self.formatter = logging.Formatter('[%(asctime)s] - %(levelname)s: %(message)s', '%d/%m/%y %H:%M:%S')
+
+    def start_streaming_the_log(self):
+
+        logging.basicConfig(level=logging.DEBUG)
```

Comment:
```suggestion
        logging.basicConfig(level=logging.INFO)
```

O Sonar captou isso como um problema de segurança. Muda o nível para INFO para evitar esse problema.